techniques/system/common/1.0/update.st

```diff
@@ -223,7 +223,7 @@ bundle agent update_action
         touch      => "true";
 
     # Copy the tools only if the file rudder_tools_updated is not up to date
-    any::
+    !policy_server::
```

Comment:
This change needs to be reflected in the reporting section below - currently, there will be no success report on root and relay servers!